renderer/components/nettests/middleboxes/HttpHeaderFieldManipulation.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { FormattedMessage } from 'react-intl'
+import { Flex, Box } from 'ooni-components'
+
+import FormattedMarkdownMessage from '../../FormattedMarkdownMessage'
+
+const HttpHeaderFieldManipulation = ({isAnomaly, render}) => {
+
+  const HHFMDetails = () => (
+    <Box>
+      <Flex my={3} flexDirection='column'>
+        {isAnomaly ? (
+          <Box>
+            <FormattedMarkdownMessage
+              id='TestResults.Details.Middleboxes.HTTPHeaderFieldManipulation.Found.Content.Paragraph'
+            />
+          </Box>
+        ): (
+          <Box>
+            <FormattedMessage
+              id='TestResults.Details.Middleboxes.HTTPHeaderFieldManipulation.NotFound.Content.Paragraph'
+            />
+          </Box>
+        )}
+      </Flex>
+    </Box>
+  )
+
+  const heroTitle = isAnomaly ? (
+    <FormattedMessage id='TestResults.Details.Middleboxes.HTTPHeaderFieldManipulation.Found.Hero.Title' />
+  ) : (
+    <FormattedMessage id='TestResults.Details.Middleboxes.HTTPHeaderFieldManipulation.NotFound.Hero.Title' />
+  )
+
+  return (
+    <div>
+      {render({
+        heroTitle: heroTitle,
+        details: <HHFMDetails />
+      })}
+    </div>
+  )
+}
+
+HttpHeaderFieldManipulation.propTypes = {
+  measurement: PropTypes.object,
+  isAnomaly: PropTypes.bool,
+  render: PropTypes.func
+}
+
+export { HttpHeaderFieldManipulation }
+
+export default {
+  name: <FormattedMessage id='Test.HTTPHeaderFieldManipulation.Fullname' />
+}
```

Comment:
Does this mean that when we import this module, if we don't specify a name to import from the module, we get by default this specific formatted message?

Reply:
Yes. This allows us to [use this pattern](https://github.com/ooni/probe-desktop/blob/ux/middlebox-measurements/renderer/components/measurement/MeasurementContainer.js#L21-L24).